api/api.swagger.json

```diff
@@ -3815,6 +3815,7 @@
     },
     "/ns/applications/{end_device_ids.application_ids.application_id}/devices/{end_device_ids.device_id}": {
       "get": {
+        "summary": "GenerateDevAddr requests a device address assignment from the Network Server.",
```

Comment:
Wonder why this was not generated before? :thinking: @htdvisser any idea?

Reply:
Strange indeed. No idea.